examples/ublox-tui/Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ["Andrei Gherghescu <andrei-ng@protonmail.com>"]
 description = "A TUI for visualizing PVT and other UBX messages"
 name = "ublox-tui"
 publish = false
-rust-version.workspace = true
+# rust-version.workspace = true
```

Comment:
why comment out this?

Reply:
It is back. I was fixing tui example, and then I forgot that one in.